taskcluster/ci/github-release-components/kind.yml

```diff
@@ -2,18 +2,21 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ---
-loader: android_taskgraph.loader.all_dep:loader
+loader: android_taskgraph.loader.multi_dep:loader
 
 transforms:
+    - android_taskgraph.transforms.multi_dep:transforms
     - android_taskgraph.transforms.treeherder:transforms
-    - android_taskgraph.transforms.all_dep:transforms
     - android_taskgraph.transforms.github_release:transforms
     - taskgraph.transforms.task:transforms
 
 kind-dependencies:
     - post-beetmover
 
+primary-dependency: post-beetmover
```

Comment:
Quick sanity check: What does `taskgraph target-graph -p taskcluster/test/params/main-repo-ship-release.yml -J`[1] show for the dependencies of `github-release-components`? I guess it should just depend on the `post-beetmover` tasks which sounds good to me.

I'd just be better safe than sorry on this because we changed the `loader` and that's the kind of issues we may find out several days after we merged it.

[1] `-J` outputs all the tasks definitions in JSON format

Reply:
Yes, just `post-beetmover`.